styles.css

```diff
@@ -28,12 +28,23 @@ aside{
   padding:1em;
   border: 5px solid #aaa;
 }
-img{
+img {
   max-width:100%;
 }
+main {
+  float:left;
+  width:50%;
+  background:#ccc;
+  padding:1em;
+  border: 5px solid #aaa;
+  order: 2;
+}
 
 @media(max-width:600px){
   .main{
     background:red;
```

Comment:
You need to use Flexbox's `order` property somewhere in here to change the order of elements when viewing this page on a mobile device.